tools/README.md

```diff
@@ -0,0 +1,151 @@
+# Python Tools
+## Pre-requisites
+### Enable python binding
+* Enable MLIR python binding, [README](https://github.com/intel/graph-compiler/blob/main/python/README.md)
+* Install `/tools/requirements.txt`
+### Set env
+* **PYTHONPATH**=*${BUILD_DIR}*/python_packages/gc_mlir_core
+* **LD_PRELOAD**=path/to/libiomp5.so
+* **MLIR_C_RUNNER_UTILS**=*${LLVM_INSTALL_DIR}*/lib/libmlir_c_runner_utils.so
+* **MLIR_RUNNER_UTILS**=*${LLVM_INSTALL_DIR}*/lib/libmlir_runner_utils.so
```

Comment:
@xurui1995 
We may need an elegant way to handle this dependency?
Copying the related shared object to the `build` directory during the `cmake` step is another option?

Reply:
@kurapov-peter @WangJialei-A  thanks for the advice, I will handle this in the next PR for migrating the code here into about the benchgc folder  https://github.com/intel/graph-compiler/issues/172,

Reply:
fixed now, get the path from `MLIR_DIR` by cmake